examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -0,0 +1,282 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+
+// vecmem
+#include <vecmem/memory/cuda/managed_memory_resource.hpp>
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// algorithms
+#include "clusterization/clusterization_algorithm.hpp"
+#include "cuda/track_finding/seeding_algorithm.hpp"
+#include "track_finding/seeding_algorithm.hpp"
+
+// io
+#include "io/csv.hpp"
+#include "io/reader.hpp"
+
+int seq_run(const std::string& detector_file, const std::string& cells_dir,
+            unsigned int skip_events, unsigned int events, bool skip_cpu,
+            bool skip_write) {
+    auto env_d_d = std::getenv("TRACCC_TEST_DATA_DIR");
+    if (env_d_d == nullptr) {
+        throw std::ios_base::failure(
+            "Test data directory not found. Please set TRACCC_TEST_DATA_DIR.");
+    }
+    auto data_directory = std::string(env_d_d) + std::string("/");
```

Comment:
There is a helper function in [utils](https://github.com/acts-project/traccc/blob/556542b39feef4b3cda06d44f2becb6adc7fda0f/io/include/io/utils.hpp#L8) that handles the data_directory.

Reply:
Fixed it now